docs/getting-started.md

```diff
@@ -31,9 +31,21 @@ If these don't work, more info is in [#805](https://github.com/StackExchange/dns
 
 Get prebuilt binaries from [GitHub releases](https://github.com/StackExchange/dnscontrol/releases/latest)
 
-Alternatively, on Mac you can install it using homebrew:
+## Through a package manager
 
-`brew install dnscontrol`
+Alternatively, on macOS (or Linux) you can install it using [Homebrew](https://brew.sh) or [MacPorts](https://www.macports.org):
```

Comment:
Binaries are also available for Linux (binary, tar, RPM, DEB), FreeBSD (tar), Windows (exec, ZIP) for 32-bit, 64-bit, and ARM at https://github.com/StackExchange/dnscontrol/releases

Could you somehow work that into the doc?

Reply:
Yep, included in the last commit.